src/js/components/Person.js

```diff
@@ -50,8 +56,14 @@ export default class Person extends Component {
     if (err) {
       this.setState({person: {}, error: err});
     } else if (res.ok) {
-      var result = res.body;
-      this.setState({person: result[0], error: null});
+      const result = res.body;
+      const person = result[0];
+
+      this.setState({person: person, error: null}, () => {
```

Comment:
Why are we waiting to get the location until after the state has updated? I think you can just call this._getLocation directly.